glean-core/python/setup.py

```diff
@@ -71,12 +71,12 @@
 version = "35.0.0"
 
 requirements = [
-    "cffi>=1",
+    "cffi>=1.13.0",
```

Comment:
If we need to, there's some rather [tricky syntax](https://www.python.org/dev/peps/pep-0508/#environment-markers) to say, for Python >= 3.8, use cffi >= 1.13.0, otherwise cffi >= 1.11.5.  But not worth the complexity unless that's an issue for our downstream consumers, esp. mach.

Reply:
I'd rather have one minimum version.
Otherwise we expand what we need to support even further for no real benefit